pkg/config/containers.conf

```diff
@@ -376,6 +376,10 @@ default_sysctls = [
 #
 #events_logger = "journald"
 
+#Sets the maximum size for event logfile in bytes
```

Comment:
Same changes as above.

Reply:
Needs a space between.  # Sets the maximum ...